general_rules/Procedure.tex

```diff
@@ -160,17 +160,21 @@ \subsection{Operator}
 
 \subsection{Moderator}
 \label{rule:moderator}
+The LOC is responsible of organizing test moderation in the local language.
 \begin{enumerate}
-	\item \textbf{Providing a moderator:} For each regular test (i.e., not for the open demonstrations), all participating teams need to provide a team member as moderator for the duration of their performance. 
+	\item \textbf{Providing a moderator:}
+	Upon OC request, a participating team may have to provide a team member as moderator for a regular test (i.e., not for the open demonstrations).
+	Candidates have to be fluent in both, English and at least one of the main languages spoken by the audience.
 	\item \textbf{Responsibilities:} The moderators have to:
 	\begin{compactitem}
-		\item explain the rules of the test, 
-		\item comment on the performance of their team, 
-		\item not interfere with the performance, 
-		\item speak in English, 
+		\item explain the rules of the test,
+		\item comment on the performance of their team,
+		\item not interfere with the performance,
+		\item speak in English and the local language,
```

Comment:
I'd change this to 'prefereably the local language' for when RoboCup is in countries with a language that is not that common. E.g when RoboCup is in the Netherlands again, all the moderation would have to be done by the potentially the single Dutch team. Moreover, lots of Dutch people are proficient enough in English.

Reply:
I must say, unlike other countries, **all** Dutch people is proficient in English, and sometimes better English speakers than some natives.

```\item speak in English and, preferably, the local language,```

Is that good enough?

Reply:
Perfect for me.